frontend/src/pages/signup/hooks/useUserIdDuplicateCheck.ts

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react';
+
+import { posValidateId } from '../apis/postValidateId';
+
+import { useConfirmState } from './useConfirmStatus';
+
+interface useDuplicateCheckParams {
```

Comment:
p3: 훅과 interface 명을 맞추면 될 거 같아 ㅎㅎ

```suggestion
interface useUserIdDuplicateCheckParams {
```

Reply:
[refactor: 훅과 interface 명을 맞게 변경](https://github.com/woowacourse-teams/2025-Fit-toring/pull/274/commits/7f63237386459ded15538bf5698bf6f2b85e0cc9)